src/test/kotlin/lotto/WinningLottoTest.kt

```diff
@@ -0,0 +1,26 @@
+package lotto
+
+import io.kotest.core.spec.style.StringSpec
+import io.kotest.matchers.shouldBe
+import lotto.domain.LottoNumber
+import lotto.domain.WinningLotto
+
+class WinningLottoTest : StringSpec({
+    "일치하는 로또번호의 개수를 확인할 수 있다." {
+        val lotto = createLotto(1, 2, 3, 4, 5, 6)
+        val bonusNumber = LottoNumber(7)
+
+        val winningLotto = WinningLotto(lotto, bonusNumber)
+
+        winningLotto.countMatchingNumbers(lotto) shouldBe 6
```

Comment:
```suggestion
        // given
        val lotto = createLotto(1, 2, 3, 4, 5, 6)
        val bonusNumber = LottoNumber(7)
        val winningLotto = WinningLotto(lotto, bonusNumber)
        
        // when
        val result = winningLotto.countMatchingNumbers(lotto) 
        result shouldBe 6
```

개인적으로 테스트 코드 작성시에, given when then 을 분리해서 작성을 하는편이예요! 
무엇을 테스트 할지, 무엇을 확인하는지 명확해지는 장점이 있어요!